frontend/lib/service-definitions/index.ts

```diff
@@ -0,0 +1,58 @@
+import groupBy from 'lodash.groupby'
+// load using js-yaml-loader
+import definitions from '../../../service-definitions.yml'
+
+export interface Category {
+  id: string
+  name: string
+}
+
+export interface Example {
+  title: string
+  example: {
+    pattern: string
+    namedParams: { [k: string]: string }
+    queryParams: { [k: string]: string }
+  }
+  preview: {
+    label?: string
+    message: string
+    color: string
+    style?: string
+    namedLogo?: string
+  }
+  keywords: string[]
+  documentation?: {
+    __html: string
+  }
+}
+
+export interface ModernRoute {
```

Comment:
Thoughts on just calling this `Route` since the other has the `Legacy` prefix?

Reply:
Works for me!